compair/api/users.py

```diff
@@ -377,6 +379,12 @@ def get(self):
                         Course.year.like(search),
                         Course.term.like(search)
                     ))
+
+        if params['includeSandbox'] == "false":
```

Comment:
can you make this flexible so like so:
```
if params['sandbox']:
    query = query.filter(Course.sandbox=params['sandbox'])
```

This allow sandbox to not effect course listing by default and also allow you to display only sandbox courses and non-sandbox courses